### PR TITLE
[BSO] Create bucket of sand

### PR DESCRIPTION
**Pull request by 6enny6**
Added create bucket of sand
uses 1 sand and 1 bucket
Added empty bucket pack to buyables 

Gives the endless sand gathered from geckos a use

### CODE REVIEW

File: src/lib/data/buyables/buyables.ts [hunk 1]
```diff
@@ -653,6 +653,13 @@ const Buyables: Buyable[] = [
 		aliases: ['wool ball', 'ball wool'],
 		gpCost: 300
 	},
+	{
+		name: 'Empty bucket pack',
+		outputItems: {
+			[itemID('Bucket')]: 100
+		},
+		gpCost: 1000
```
Comment (Lajnux) [inline src/lib/data/buyables/buyables.ts:661]: Slightly too close price to in game GE leaving it vulnerable for a temporary flux over 10 gp and therefor might result in infinit GP. 
 ```suggestion
		gpCost: 10_000
```

File: src/lib/data/buyables/buyables.ts [hunk 2]
```diff
@@ -653,6 +653,13 @@ const Buyables: Buyable[] = [
 		aliases: ['wool ball', 'ball wool'],
 		gpCost: 300
 	},
+	{
+		name: 'Empty bucket pack',
+		outputItems: {
+			[itemID('Bucket')]: 100
+		},
+   	gpCost: 10_000
```
Comment (Lajnux) [inline src/lib/data/buyables/buyables.ts:661]:
```suggestion
   	gpCost: 10_000
```

Comment (Lajnux) [inline src/lib/data/buyables/buyables.ts:661]:
```suggestion
    gpCost: 10_000
```

File: src/lib/data/buyables/buyables.ts [hunk 3]
```diff
@@ -653,6 +653,13 @@ const Buyables: Buyable[] = [
 		aliases: ['wool ball', 'ball wool'],
 		gpCost: 300
 	},
+	{
+		name: 'Empty bucket pack',
+		outputItems: {
+			[itemID('Bucket')]: 100
+		},
+    gpCost: 10_000
```
Comment (Lajnux) [inline src/lib/data/buyables/buyables.ts:661]:
```suggestion
        gpCost: 10_000
```

File: src/lib/data/buyables/buyables.ts [hunk 4]
```diff
@@ -653,6 +653,13 @@ const Buyables: Buyable[] = [
 		aliases: ['wool ball', 'ball wool'],
 		gpCost: 300
 	},
+	{
+		name: 'Empty bucket pack',
+		outputItems: {
+			[itemID('Bucket')]: 100
+		},
+      gpCost: 10_000
```
Comment (Alexsuperfly) [inline src/lib/data/buyables/buyables.ts:661]:
```suggestion
		gpCost: 10_000
```

**Pull request merged**